source/2019-12-19-countdown-to-the-new-year-ember-template-lint.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Countdown to The New Year - Ember Template Lint
+author: Melanie Sumner
+tags: 2019, Accessibility, Addons, DecEmber
+alias: 2019-12-19-countdown-to-the-new-year-ember-template-lint.md
+responsive: true
+---
+
+**This is the 19th in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 19
+
+Today we are going to look at an addon that, while it is built into Ember apps by default, has had enough updates lately to deserve a closer look- [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint).
+
+READMORE
+
+### What It Does
+
+As you may guess from the addon title, `ember-template-lint` is a linter for your template files! 
+Because we work in JavaScript, there are different ways something *could* be done- but often there is an ideal solution that we know won't cause problems down the line. 
```

Comment:
```suggestion
Because we work in JavaScript, there are different ways something *could* be done, but often there is an ideal solution that avoids problems down the line. 
```